_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,353 @@
+    other contract protocols.
+
+    The authors of the paper propose a solution they call Mutually
+    Assured Destruction HTLCs (MAD-HTLCs).  These require Bob to provide a
+    hashlocked bond that makes him reveal his own preimage when sending
+    his timeout settlement.  If both Alice and Bob reveal their
+    respective preimages, miners will be able to claim both the
+    payment/refund amount and the bond collateral---destroying the
+    incentive for Bob to attempt stealing by bribing miners.
+
+    Downsides to this approach are that it requires Bob to use more
+    collateral---raising the cost of using HTLCs---and it uses more
+    block chain space when MAD-HTLCs are settled onchain compared to
+    traditional HTLCs---also raising costs.  It was also
+    [claimed][harding mad miner] in the mailing list discussion that
+    MAD-HTLCs might have their own problems with theft when the
+    bonded user's counterparty is a miner (e.g. Alice is a miner).
+
+    ZmnSCPxj [noted][zmnscpxj scorched earth] that there already exists
+    a mechanism to allow Alice to discourage Bob from attempting his
+    theft: Alice can enact a [scorched earth][] policy where she spends
+    all of her legitimate funds to fees in order to prevent Bob from
+    receiving them.  In theory, this should prevent Bob from even trying
+    the theft.  It was also [explained][harding myopic] that nearly all
+    miners would need to switch to the proposed bribe-accepting software
+    before the attack would become particularly effective.
+
+    In the short term, and probably the medium term, this attack does
+    not appear to pose any significant danger to HTLCs whose receivers
+    resolve their preimages well before their timelocks expire.  In the
+    long term, this is an incentive compatibility concern that
+    protocol developers may want to keep in mind.
+
+- **Proposed service for storing, relaying, and broadcasting presigned transactions:**
+  Jacob Swambo sent a [request for comments][swambo rfc] to the Bitcoin-Dev
+  mailing list about creating software and a protocol for allowing
+  applications to store presigned transactions with third-party services
+  for later broadcast.  The software could also perhaps determine when to
+  broadcast the transactions based on certain conditions being met.  This
+  could be useful for software such as [vaults][topic vaults] and
+  [watchtowers][topic watchtowers].  Swambo asks anyone interested in
+  the idea, especially those interested in using it with their protocol, to
+  contact him.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **Magical Bitcoin:** Alekos Filini presented at LA BitDevs on [Magical
+  Bitcoin][], an in-development set of tools and libraries for onchain
```

Comment:
perhaps s/an in-development set of tools and libraries/a set of tools and libraries under development/